docs/index.rst

```diff
@@ -0,0 +1,212 @@
+Picobox
+=======
+
+.. image:: _static/picobox.svg
+   :width: 200
+   :align: right
+
+Picobox is opinionated `dependency injection`__ framework designed to be clean,
+pragmatic and with Python in mind. No complex graphs, no implicit injections,
+no type bindings, no XML configurations.
+
+.. __: https://en.wikipedia.org/wiki/Dependency_injection
+
+
+Why?
+----
+
+Dependency Injection (DI) design pattern is intended to decouple various parts
+of an application from each other. So a class can be independent of how the
+objects it requires are created, and hence the way we create them may be
+different for production and tests.
+
+One of most easiest examples is to say that DI is essentially about writing
+
+.. code:: python
+
+    def do_something(my_service):
+        return my_service.get_val() + 42
+
+    my_service = MyService(foo, bar)
+    do_something(my_service)
+
+instead of
+
+.. code:: python
+
+    def do_something():
+        my_service = MyService(foo, bar)
+        return my_service.get_val() + 42
+
+    do_something()
+
+because the latter considered non-configurable and is harder to test.
+
+In Python, however, dependency injection is not a big deal due to its dynamic
+nature and duck typing: anything could be defined anytime and passed anywhere.
+Due to that reason (and maybe some others) DI frameworks aren't popular among
+Python community, though they may be handy in some cases.
+
+One of such cases is code decoupling when we want to create objects in one
+places and use them in others, preserving clean interface and avoiding global
```

Comment:
I get what you mean, but since "one" in English doesn't really have plural form (unlike «одних»), "one places" feels like a conflict of singular and plural.

I'd either replace it with "create objects in some places and use them in the others" (which has the closest semantic meaning to the original), or just rephrase it, like "create and use objects in different places".